### PR TITLE
fix(validate): surface all validation errors

### PR DESCRIPTION
**Pull request by erunion**
| 🚥 Fix https://github.com/readmeio/rdme/issues/638 |
| :-- |

## 🧰 Changes

This updates our [@readme/openapi-parser](https://npm.im/@readme/openapi-parser) and [oas-normalize](https://npm.im/oas-normalize) dependencies to begin surfacing **all** validation errors in OpenAPI and Swagger validation.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,8 +50,8 @@
     "jsonpath": "^1.1.1",
     "mime-types": "^2.1.35",
     "node-fetch": "^2.6.1",
-    "oas": "^18.4.4",
-    "oas-normalize": "^7.0.0",
+    "oas": "^19.0.1",
```
Comment (erunion) [inline package.json:53]: Nothing new there that's going to be used within the library, I just released a v19 last week so might as well update it here.

**Review by kanadgupta — APPROVED** [review]
Would you consider this a breaking change? If so, could you rename the PR title from `fix(validate)` to `feat(breaking/validate)`?

**Comment by erunion** [comment]
@kanadgupta No I wouldn't consider this a breaking change.

File: package.json (hunk 1 above)
Comment (kanadgupta) [reply package.json:53]: Do we even use `oas` anywhere? Nothing comes up during my quick search

Comment (kanadgupta) [reply package.json:53]: Ignore me, it's used in the reducer.

Comment (erunion) [reply package.json:53]: We load it in for the reducer command as that code is in `oas`.

**Pull request merged**